lms/envs/common.py

```diff
@@ -1355,15 +1355,17 @@
     },
     'style-certificates': {
         'source_filenames': [
-            'certificates/css/main-ltr.css',
+            'sass/certificates/main-ltr.css',
```

Comment:
@talbs yes you would just revert this change: https://github.com/edx/edx-platform/commit/b0dbe2f2dab908f0a12dbce121a43ea24fff609d#diff-b44ddc0cc763f679e4e1626a21d24e73